### ISSUE TITLE
small number of n

### ISSUE THREAD

**Issue by ak9250** [NVlabs/stylegan2-ada#20]
for n less than a 100 did anyone experience mode collapse?

**Comment by YukiSakuma** [comment]
It happened to me, has anyone managed to successfully train with less than 100 images or even less than 50 with stylegan2-ada?

**Comment by tr1pzz** [comment]
I've had somewhat succeful runs with ~300 images using:

- Finetuning from a good pretrained model (StyleGAN2 trained on ImageNET)
--freezed=12
- Very small learning rate (0.0005)

The samples look OK, but the diversity is rather low... But that's what you'd expect I guess...

**Comment by woctezuma** [comment]
>
>     * Finetuning from a good pretrained model (StyleGAN2 trained on ImageNET)
>       --freezed=12

Which pre-trained model is it?

I cannot find it here: https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada/pretrained/transfer-learning-source-nets/

Is it this? https://twitter.com/theshawwn/status/1243271022616297472?s=20

> 
>     * Very small learning rate (0.0005)
> 

Interesting. SO it is 4 times less than the default value of the `auto` config.

https://github.com/NVlabs/stylegan2-ada/blob/8c3fd8bac3e5a54a20e860fc163d6e67cc03c623/train.py#L164
